bridge/irc_manager.go

```diff
@@ -90,10 +90,17 @@ func (m *IRCManager) DisconnectUser(userID string) {
 
 var connectionsIgnored = 0
 
+func (m *IRCManager) ircIgnoredDiscord(user string) bool {
+	return m.bridge.Config.IRCIgnoresDiscord[strings.ToLower(user)]
+}
+
 // HandleUser deals with messages sent from a DiscordUser
 //
 // When `user.Online == false`, we make `user.ID` the only other data present in discord.handlePresenceUpdate
 func (m *IRCManager) HandleUser(user DiscordUser) {
+	if m.ircIgnoredDiscord(user.Nick) {
```

Comment:
instead of matching by nickname, i think we should match by `user.ID`

and we could call the yaml setting `ignored_discord_user_ids`

Reply:
We probably should indeed, just when I first wrote the code I did it with nickname matches. I'll see if after I get the other feature branches submitted that I can rework it for `user.ID` instead.

Reply:
This part is done in both places that call is issued now.